.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+
+## Initial triage
+When sitting down to do some triaging work, start with the list of untriaged tickets. Consider all tickets that do not have a label as untriaged. The first step is to categorize the ticket into one of the following categories and either close the ticket or assign an appropriate lable. The reported issue
+
+ *  is not valid
+If you think the ticket is invalid comment why you think it is invalid, then close the ticket. Tickets might be invalid if they were already fixed in the past or it was decided that the proposed feature will not be implemented because it does not conform with the overall goal of Pocket-PiAI Project. Also if you happen to know that the problem was already reported, label the ticket with Status: duplicate, reference the other ticket that is already addressing the problem and close the duplicate.
+
+Examples:
+
+This is an issue. (A good starting point for first person to discover)
+
+
+ *  does not provide enough information
+Add the label question if the reported issue does not contain enough information to decide if it is valid or not and ask on the ticket for the required information to go forward. I will re-triage all tickets that have the label question assigned. If the original reporter left new information we can try to re-categorize the ticket. If the reporter did not come back to provide more required information after a long enough time, we will close the ticket (this should be roughly about two weeks).
+
+Examples:
+
+My builds stopped working. Please help! Ask for a link to the build log and for which project is affected.
+
+
+ *  is a valid enhancement proposal
+If the ticket contains an enhancement proposal that aligns with the goals of this repo, then add the label Enhancement. If the proposal seems valid but requires further discussion between core contributors because there might be different possibilities on how to implement the enhancement, then also add the label question.
+
+Examples:
+
+Improve documentation Examples in contribute docs.
+Provide better integration with security feature XYZ
+Refactor module X for better readability (see #48)
+Achieve world domination (also needs the label question)
+
+
+ *  is a valid problem within the code base:
+If it’s a valid bug, then add the label Bug. Try to reference related issues if you come across any.
+
+Examples:
+
+Example Library fails to transmit the leter 'x' or 'y' (logs attached)
+
+
+ *  is a question and needs answering:
+If the ticket contains a question about the code, then move question to FAQ on wiki and add task to answer question on wiki to the issue (or close if already answered).
+
+Examples:
+
+My message was seen by two hosts. Why?
+Why are my builds failing?
+
+
+Helpful links for triaging
+Here is a list of links for contributors that look for work:
+
+Untriaged tickets: Go and triage them!
+
+
```

Comment:
:-1: This needs reworked to be more of a template than it is. remove references to other projects for example.